examples/in_toto_run_test.rs

```diff
@@ -16,6 +16,7 @@ fn main() {
         Some(&["sha512", "sha256"]),
     )
     .unwrap();
+    let json = serde_json::to_value(&link).unwrap();
 
-    println!("Generated link: {:?}", link)
+    println!("Generated link: {}", json)
```

Comment:
My reasoning at the time is that this is a link but converted into the json format, and since we have `Link` as a struct, it made more sense to name this a json.